Audio/JammitFolderSongPlayer.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Jammit.Model;
+using NAudio.Wave;
+
+namespace Jammit.Audio
+{
+  /// <summary>
+  /// Handles playing the audio from a JCF track package (folder).
+  /// </summary>
+  public class JammitFolderSongPlayer : ISongPlayer
+  {
+    private readonly List<WaveChannel32> _channels;
+    private readonly WaveMixerStream32 _mixer;
+    private readonly WaveOutEvent _waveOut;
+    private readonly List<string> _chanNames;
+
+    /// <summary>
+    /// Creates a player instance for the giving song.
+    /// </summary>
+    /// <param name="s">FolderSong to play.</param>
+    internal JammitFolderSongPlayer(FolderSong s)
+    {
+      _waveOut = new WaveOutEvent();
+      _mixer = new WaveMixerStream32();
+      _channels = new List<WaveChannel32>();
+      _chanNames = new List<string>();
+
+      foreach(var t in s.Tracks)
+      {
+        if (t.ClassName == "JMFileTrack")
+        {
+          using (var fs = new FileStream($"{Properties.Settings.Default.TrackPath}/{s.Metadata.GuidString}.jcf/{t.Id}_jcfx", FileMode.Open))
```

Comment:
This should use the new SongPath field I mention in the other comment: `$"{s.SongPath}/{t.Id}_jcfx"`

Also, I think we could move audio file loading to the ISong interface, returning a stream to the .aifc file. But that can be done later.